erp/schema.py

```diff
@@ -642,17 +645,30 @@
             PERSONNELS_AUCUN,
         ],
     },
-    "accueil_equipements_malentendants": {
+    "accueil_equipements_malentendants_presence": {
         "is_a11y": True,
-        "label": "Équipement(s) de communication",
+        "label": "Équipement(s) de communication présence",
```

Comment:
Unfortunate wording 
![image](https://user-images.githubusercontent.com/41547/106446273-e780ff80-6434-11eb-97f4-121d984c11fc.png)

Reply:
Also for the equipement choices help text, maybe *Sélectionnez les équipements sourds/malentendants disponibles*